ee/clickhouse/models/cohort.py

```diff
@@ -163,7 +163,7 @@ def get_entity_cohort_subquery(cohort: Cohort, cohort_group: Dict, group_idx: in
         return f"distinct_id IN ({extract_person})", {**entity_params, **date_params}
 
 
-def _get_count_operator(count_operator: Optional[str]) -> str:
+def get_count_operator(count_operator: Optional[str]) -> str:
```

Comment:
this is a bit annoying, as the property operator and cohort count operator are sadly disjoint

(property operator is `gt`, cohort operator is `gte`. Let's reconcile!